metabase/metabase
# Display official badge throughout the application

We want to display an official badge for questions and dashboards in official collections.
This PR adds official badges to:

* Collections sidebar
* Question page
* Dashboard page
* Search listings (for a collection itself and collection contents)
* Saved question picker
* Collection picker (the one you see when e.g. saving a new question or dashboard)
* Question picker (the one you see in a sidebar when adding new questions to dashboards)

### To Verify

1. Spin up Metabase Enterprise and sign in as admin
2. Go to `/collection/root`
3. Click a folder item in the top right ("New collection")
4. Select the "Official" collection type, save the collection
5. Move a question and a dashboard to it
6. Check different places from the demo section below and make sure it looks the same. For regular items (saved to regular collections) it should behave the same way it does on `master`

### Demos

<details>
<summary>Video (demos every UI change with voiceover)</summary>

https://user-images.githubusercontent.com/17258145/126348493-fbaca4d2-e4ee-4f65-8f8b-359134b3a840.mov

</details>

<details>
<summary>Collection sidebar</summary>

![collections-sidebar](https://user-images.githubusercontent.com/17258145/126349394-d1c1058f-998a-41d2-853a-482b49a7eab3.png)

</details>

<details>
<summary>Question page</summary>

![question-page](https://user-images.githubusercontent.com/17258145/126350003-a91d7729-2ece-4fea-a920-02547769e066.png)

</details>

<details>

<summary>Dashboard page</summary>

![dashboard-page](https://user-images.githubusercontent.com/17258145/126349451-1fbe5a7e-269c-45fd-9a5a-521899d5dab1.png)

</details>

<details>
<summary>Search</summary>

![search](https://user-images.githubusercontent.com/17258145/126349741-10c2bb1e-65a1-491b-bb7a-43425f08a7b9.png)

</details>

<details>
<summary>Saved question picker</summary>

![saved-question-picker](https://user-images.githubusercontent.com/17258145/126349813-93860070-2f1c-4d25-a1de-b198000d9a6a.png)

</details>

<details>
<summary>Collection picker</summary>

![collection-picker](https://user-images.githubusercontent.com/17258145/126349854-1bc2a84d-6272-4f2e-ad8d-1cd924334165.png)

</details>

<details>
<summary>Question picker</summary>

![question-picker](https://user-images.githubusercontent.com/17258145/126349888-faa5726e-3180-4be9-aa95-f91e2e9674d0.png)


</details>
